### DISCUSSION TITLE
react-dom.8af4f3a1.js:9 Uncaught TypeError: Cannot read properties of undefined (reading '__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED')     at react-dom.8af4f3a1.js:9:5345

### DISCUSSION THREAD

**Discussion by limuen** [vercel/community#1649] [Help]
### Summary

react-dom.8af4f3a1.js:9 Uncaught TypeError: Cannot read properties of undefined (reading '__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED')
    at react-dom.8af4f3a1.js:9:5345

**Comment by swarnava** [comment]
What is the issue you are facing?

**Reply by limuen** [reply]
When the project runs locally, it is OK to push it to github. After varcel deploys and opens the url address, the console will report this error

**Reply by swarnava** [reply]
Please share a [minimal repository](https://stackoverflow.com/help/minimal-reproducible-example) so we can investigate.

**Reply by limuen** [reply]
https://react-arco-pro.vercel.app/login 
repository: https://github.com/limuen/React-Arco-Pro
I have uploaded a basic template now, and now 404
